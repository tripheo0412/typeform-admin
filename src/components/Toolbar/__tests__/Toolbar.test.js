import React from 'react';

import { shallow, mount } from 'enzyme';

import { Toolbar } from '..';

describe('Toolbar', () => {
  let wrapper;
  it('Toolbar should render correctly', () => {
    wrapper = shallow(<Toolbar />);
    expect(wrapper).toMatchSnapshot();
  });
  it('Toolbar should render correctly the theme variant', () => {
    wrapper = mount(<Toolbar variant="theme" />);
    expect(wrapper.find('.container--theme')).toHaveLength(1);
  });
  it('Toolbar variant prop should be main by default', () => {
    wrapper = mount(<Toolbar />);
    expect(wrapper.props().variant).toBe('main');
    expect(wrapper).toMatchSnapshot();
  });
});
