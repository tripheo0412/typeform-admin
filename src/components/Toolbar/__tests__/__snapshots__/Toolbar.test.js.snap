// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Toolbar should render correctly 1`] = `ShallowWrapper {}`;

exports[`Toolbar Toolbar variant prop should be main by default 1`] = `ReactWrapper {}`;
